src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -4752,6 +4756,13 @@ def is_exactly_one_true(*args):
     return found
 
 
+def list_flexconsumption_zone_redundant_locations(cmd):
+    client = web_client_factory(cmd.cli_ctx)
+    regions = client.list_geo_regions(sku="FlexConsumption")
```

Comment:
What happens when this API call fails?  Does the operation fail or the method return no locations?

Reply:
AZ and SDK surfaces all API errors to the users.